janitor/site/pkg.py

```diff
@@ -375,7 +375,7 @@ async def generate_done_list(
         runs.append(run)
 
     return {
-        "oldest": oldest, "runs": runs, "campaign": campaign,
+        "oldest": oldest, "runs": runs, "suite": suite, "campaign": campaign,
```

Comment:
Yes there, but the `suite` is used in navigation sidebar template. Then should I check all endpoints return `campaign` instead?

Reply:
Yeah, that's eventual goal. I'd also be happy to merge something that sets "suite" to campaign in generate_done_list() for now, if you don't want to make that larger change.

Reply:
OK, will replace this PR with adding `campaign` where necessary + replacing `suite` with `campaign` in site templates and add a followup PR in salsa.

Reply:
Also please have a look at https://janitor.debian.net/fresh-releases/ and https://code.breezy-vcs.org (namely brz-debian)

Reply:
Any news on this?

Reply:
I still have to distill the k8s of Debian Janitor into our single-node pod first. But I am going to complete this task as well. Even more, database sql changes need migration functionality and branch override feature is also what I would need to spin NeuroDebian Janitor up.

Reply:
Yeah, at the moment the janitor core is still very in flux so maintaining an instance is quite labor-intensive. The boundary between the core and the instances also isn't well flushed out yet and still changing.

If you're planning to run your instance, it might make sense for us to define what the expected interface between core and instances is like.